### PR TITLE
ignore preflights and deploy application

### PR DESCRIPTION
**Pull request by stefanrepl**
#### What type of PR is this?
type::feature

#### What this PR does / why we need it:
Deploy application when a user chooses to ignore preflights and deploy.  We need this because the previous flow of choosing ignore preflights and continue was a confusing for some users as they expected their app to deploy when choosing this option.

#### Which issue(s) this PR fixes:

Fixes #
https://app.shortcut.com/replicated/story/36852/skip-preflight-checks-on-initial-install-does-not-install-app

#### Special notes for your reviewer:
Ran against nightly regression tests.  Tested deploy functionality in okteto.

#### Does this PR introduce a user-facing change?

```release-note
     Automate deployment of application when preflights are ignored.
```

### CODE REVIEW

File: web/src/components/shared/modals/SkipPreflightsModal.jsx [hunk 1]
```diff
@@ -22,9 +22,9 @@ export default function SkipPreflightsModal(props) {
           <div className="u-marginTop--30 flex flex-column">
             <button type="button" className="btn blue primary" onClick={hideSkipModal}>Wait for Preflights to finish</button>
             {onForceDeployClick ?
-              <span className="u-fontSize--normal u-fontWeight--medium u-textDecoration--underline u-textColor--bodyCopy u-marginTop--15 u-cursor--pointer" onClick={() => onForceDeployClick(false)}>Ignore Preflights and deploy</span>
+              <span className="u-fontSize--normal u-fontWeight--medium u-textDecoration--underline u-textColor--bodyCopy u-marginTop--15 u-cursor--pointer" onClick={() => onForceDeployClick(true)}>Ignore Preflights and deploy</span>
```
Comment (sgalsaleh) [inline web/src/components/shared/modals/SkipPreflightsModal.jsx:25]: why was this changed to `true`? it looks like the `onForceDeployClick` takes a `continueWithFailedPreflights` and in this case they didn't fail but rather skipped, right?

Comment (stefanrepl) [reply web/src/components/shared/modals/SkipPreflightsModal.jsx]: correct, thanks, updated

**Review by sgalsaleh — APPROVED** [review]
